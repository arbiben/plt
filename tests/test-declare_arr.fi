arr int 6 a?
arr str 20 b?

int main(){

printstring(b@20)?
print(a@1)?
printstring(" ")?
printstring("bye!")?
}
