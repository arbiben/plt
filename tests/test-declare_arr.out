(null)0 bye!